code/datums/quirks/neutral.dm

```diff
@@ -173,6 +173,56 @@
 	species.disliked_food = initial(species.disliked_food)
 	UnregisterSignal(human_holder, COMSIG_SPECIES_GAIN)
 
+/datum/quirk/heterochromatic
+	name = "Heterochromatic"
+	desc = "One of your eyes is a different color than the other!"
+	icon = "eye-low-vision" // Ignore the icon name, its actually a fairly good representation of different color eyes
+	value = 0
+	var/color
+
+/datum/quirk/heterochromatic/add()
+	color = color || quirk_holder.client?.prefs?.read_preference(/datum/preference/color/heterochromatic)
+	if(!color)
+		return
+
+	link_to_holder()
+
+/datum/quirk/heterochromatic/post_add()
+	if(color)
+		return
+
+	color = quirk_holder.client?.prefs?.read_preference(/datum/preference/color/heterochromatic)
+	if(!color)
+		return
+
+	link_to_holder()
+
+/datum/quirk/heterochromatic/proc/link_to_holder()
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	human_holder.eye_color_heterochromatic = TRUE
+	human_holder.eye_color_right = color
+	RegisterSignal(human_holder, COMSIG_CARBON_LOSE_ORGAN, .proc/check_eye_removal, override=TRUE)
```

Comment:
Why `override = TRUE`? Every usage of this should be commented, otherwise it just looks like you're silencing a warning.

Also, it doesn't look like you unregister this when the quirk is removed.

Reply:
Override is set here because link to holder is called by the preference, I can add a comment for this if you think it's needed

Reply:
it is

Reply:
Done